tsdb/errors/errors.go

```diff
@@ -41,22 +80,11 @@ func (es MultiError) Error() string {
 	return buf.String()
 }
 
-// Add adds the error to the error list if it is not nil.
-func (es *MultiError) Add(err error) {
-	if err == nil {
-		return
-	}
-	if merr, ok := err.(MultiError); ok {
-		*es = append(*es, merr...)
-	} else {
-		*es = append(*es, err)
-	}
-}
-
-// Err returns the error list as an error or nil if it is empty.
-func (es MultiError) Err() error {
-	if len(es) == 0 {
-		return nil
+// CloseAll closed all given closes with all errors recorded. All closers are closed.
```

Comment:
Probably because I'm new here, but this comment is quite confusing :open_mouth:

Reply:
no it's a typo will fix (: